### PR TITLE
Add xps to pdf conversion details, PrintJob.Create and PrinterShare.ReadBasic.All scope and minor fixes

### PR DESCRIPTION
**Pull request by nilakhan**
Add xps to pdf conversion details, PrintJob.Create and PrinterShare.ReadBasic.All scope and minor fixes

### CODE REVIEW

**Review by braedenp-msft — APPROVED** [review]

File: api-reference/beta/api/printdocument-createuploadsession.md [hunk 1]
```diff
@@ -60,6 +60,11 @@ In the request body, provide a JSON object with the following parameters.
 |:-------------|:------------|:------------|
 |properties|[printDocumentUploadProperties](../resources/printDocumentUploadProperties.md)|Represents properties of the binary file to be uploaded.|
 
+The **contentType** property in the request body should be supported by the printer/printer share (defined in [printerCapabilities](../resources/printercapabilities.md)). The only exception to this is the scenario where Universal Print converts **OXPS to PDF**, which will happen if **all** the following conditions are met:
```
Comment (Lauragra) [inline api-reference/beta/api/printdocument-createuploadsession.md]: I'm not sure that this statement is entirely clear. What are we getting at with the link to printerCapabilities, vs. printDocumentUploadProperties? Can you better explain the exception? What is the action for the request if the three conditions are met?

File: api-reference/beta/api/printdocument-get-file.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Download the binary file associated with a [printDocument](../resources/printdoc
 ## Permissions
 One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
 
-In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription and have a permission that grants [Get printer](printer-get.md) access.
+In addition to the following permissions, the user or app's tenant must have an active Universal Print subscription and have either the _Printer.Read.All_ or _Printer.ReadWrite.All_ application permission.
```
Comment (Lauragra) [inline api-reference/beta/api/printdocument-get-file.md:21]: Global: Although the permission names are italicized in the permissions reference, we don't italicize them in content, in the permissions tables, etc.

File: api-reference/beta/api/printer-get.md [hunk 3]
```diff
@@ -45,6 +45,11 @@ This method supports some of the OData query parameters to help customize the re
 Do not supply a request body for this method.
 ## Response
 If successful, this method returns a `200 OK` response code and [printer](../resources/printer.md) object in the response body.
+By default, the response will not contain [printerCapabilities](../resources/printerCapabilities.md). **To get printerCapabilities, use the $select query parameter**. e.g.:
```
Comment (Lauragra) [inline api-reference/beta/api/printer-get.md]: I recommend mentioning this in the query parameters section. Here in the response section you can simply state that if the `$select` parameter is specified, the response will also include printer capabilities. Also, you might want to add an example that shows how to get a printer and its printer capabilities, and what the response looks like.

File: api-reference/beta/api/printershare-get.md [hunk 4]
```diff
@@ -48,6 +48,11 @@ This method supports some of the OData query parameters to help customize the re
 Do not supply a request body for this method.
 ## Response
 If successful, this method returns a `200 OK` response code and a [printerShare](../resources/printershare.md) object in the response body.
+By default, the response will not contain [printerCapabilities](../resources/printerCapabilities.md). **To get printerCapabilities, use the $select query option**. e.g.:
```
Comment (Lauragra) [inline api-reference/beta/api/printershare-get.md]: Same comment here as previous - add this to the query parameters section and add an example.

**Review by tomsato-ms — APPROVED** [review]

File: api-reference/beta/api/printdocument-createuploadsession.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ In addition to the following permissions, the user or app's tenant must have an
 
 | Permission type                        | Permissions (from least to most privileged) |
 |:---------------------------------------|:--------------------------------------------|
-| Delegated (work or school account)     | PrintJob.ReadWrite, PrintJob.ReadWrite.All |
+| Delegated (work or school account)     | PrintJob.Create, PrintJob.ReadWrite, PrintJob.ReadWrite.All |
```
Comment (Lauragra) [inline api-reference/beta/api/printdocument-createuploadsession.md:27]: Global: If you want to get rid of the warnings on the PR, you'll eventually want to update your ms.prod values to "cloud-printing".

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please see comments on the PR.

**Review by jahsu-MSFT — APPROVED** [review]

File: api-reference/beta/api/printdocument-get-file.md (hunk 2 above)
Comment (nilakhan) [reply api-reference/beta/api/printdocument-get-file.md]: Updated. Thanks

File: api-reference/beta/api/printdocument-createuploadsession.md (hunk 5 above)
Comment (nilakhan) [reply api-reference/beta/api/printdocument-createuploadsession.md:27]: Where can we update this?

File: api-reference/beta/api/printdocument-createuploadsession.md (hunk 1 above)
Comment (nilakhan) [reply api-reference/beta/api/printdocument-createuploadsession.md]: I have updated this documentation. Please let me know if there are any concerns.

File: api-reference/beta/api/printer-get.md (hunk 3 above)
Comment (nilakhan) [reply api-reference/beta/api/printer-get.md]: Updated. thanks!

File: api-reference/beta/api/printershare-get.md (hunk 4 above)
Comment (nilakhan) [reply api-reference/beta/api/printershare-get.md]: Updated. Thanks!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**